tests/unit/cbprintf/main.c

```diff
@@ -1141,6 +1141,56 @@ static void test_p(void)
 	zassert_equal(rc, 12, NULL);
 	zassert_equal(strncmp("/0x00cafe21/", buf, rc), 0, NULL);
 }
+static void test_alternative_form_hex(void)
```

Comment:
commit message is wrong. Should be something like:
`tests: unit: cbprintf: Add test for ...`

Also the email address...